pkg/downloader/manager.go

```diff
@@ -147,14 +150,27 @@ func (m *Manager) Update() error {
 		return nil
 	}
 
-	// Check that all of the repos we're dependent on actually exist and
-	// the repo index names.
+	// Get the names of the repositories the dependencies need that Helm is
+	// configured to know about.
 	repoNames, err := m.resolveRepoNames(req)
 	if err != nil {
 		return err
 	}
 
-	// For each repo in the file, update the cached copy of that repo
+	// For the repositories Helm is not configured to know about, ensure Helm
+	// has some information about them and, when possible, the index files
+	// locally.
+	// TODO(mattfarina): Repositories should be explicitly added by end users
+	// rather than automattic. In Helm v4 require users to add repositories. They
```

Comment:
typos in comment (automattic), "For security" vs. ", for securirty"

Reply:
Fixed... did you notice you misspelled "securirty" ;-). I find it amusing that a typo fix includes a typo.

Reply:
hahaha